content/en/docs/tasks/job/parallel-processing-expansion.md

```diff
@@ -1,50 +1,62 @@
 ---
 title: Parallel Processing using Expansions
-content_template: templates/concept
+content_template: templates/task
 weight: 20
 ---
 
 {{% capture overview %}}
 
-In this example, we will run multiple Kubernetes Jobs created from
-a common template.  You may want to be familiar with the basic,
-non-parallel, use of [Jobs](/docs/concepts/workloads/controllers/jobs-run-to-completion/) first.
+In this example, you will run multiple {{< glossary_tooltip text="Jobs" term_id="job" >}}
+based a common template. You can use this approach to process batches of work in
+parallel.
 
+For this task there are only three items: [_apple_, _banana_, _cherry_].
+The example Jobs process each item simply by printing a string then pausing.
+
+See [using Jobs in real workloads](#using-jobs-in-real-workloads) to learn about how
+this pattern fits more realistic use cases.
 {{% /capture %}}
 
+{{% capture prerequisites %}}
 
-{{% capture body %}}
+You should be familiar with the basic,
+non-parallel, use of [Job](/docs/concepts/jobs/run-to-completion-finite-workloads/).
 
-## Basic Template Expansion
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
 
-First, download the following template of a job to a file called `job-tmpl.yaml`
+You will need the command-line utility `sed`, a working install of
+[Python](https://www.python.org/), and the Jinja2 template
```

Comment:
Possibly make the prereqs into a list?

Reply:
Will that work OK with the `{{< include "task-tutorial-prereqs.md" >}}` and `{{< version-check >}}` shortcodes?

:thinking:

Reply:
How's this?

Reply:
OK